docs/conf.py

```diff
@@ -174,14 +166,13 @@
 napoleon_google_docstring = False
 
 # -- Options for the edit_on_github extension ----------------------------------------
-extensions.remove('astropy_helpers.extern.numpydoc')
 extensions.append('sphinx.ext.napoleon')
 
```

Comment:
Hey @Cadair ,Why `sphinx.ext.napoleon` is appended 2 times ,does this serve any purpose ? Also `numpydocs` should be removed from `extensions` right ?